sim_volumes.py

```diff
@@ -0,0 +1,172 @@
+"""Generate 3D map of molecules."""
+
+import numpy as np
+from geomstats.geometry.special_orthogonal import SpecialOrthogonal
+
+SO3 = SpecialOrthogonal(n=3, point_type="vector")
+
+PARTICULES = np.asarray([[1, 0, 0, 3], [0, 8, 0, 5], [0, 0, 7, 9]])
+N_VOLUMES = 2000
+VOL_SIZE = 64
+NAME = "4_points_3d"
+CENTER = 2
+
+
+def get_random_quat(num_pts):
+    """Generate a list of quaternions by using uniform distribution.
+
```

Comment:
How do you know this samples from a __uniform__ distribution, do you have a reference (Wikipedia, research paper) that explains why these formulas work?

Reply:
okay I will add that but i'm not sure how to present that into the Docstring of the function